server/middleware/workstack.js

```diff
@@ -79,12 +103,35 @@ async function allocateToTeam(req, res, next) {
             .map(([caseId, stageId]) => [`/case/${caseId}/stage/${stageId}/user`, { userUUID: selected_user }, {
                 headers: User.createHeaders(req.user)
             }])
-            .map(async options => await allocateUser(req, res, options));
+            .map(async options => await putToCaseworkService(req, res, options));
         await Promise.all(requests);
     }
     next();
 }
 
+async function moveTeam(req, res, next) {
```

Comment:
I can't see any tests for this method, may need some adding

Reply:
The tests for "handleWorkstackSubmit" cover this method when the, but I've renamed this to "workstack submit case update (allocate/move team)" as it covers the whole path.